content/docs/command-reference/checkout.md

```diff
@@ -10,21 +10,22 @@ usage: dvc checkout [-h] [-q | -v] [--summary] [-d] [-R] [-f]
                     [--relink] [targets [targets ...]]
 
 positional arguments:
-  targets        Limit command scope to these stages or .dvc files.
-                 Using -R, directories to search for stages or .dvc
-                 files can also be given.
+  targets       Limit command scope to these tracked files/directories,
+                .dvc files, or stage names.
 ```
 
 ## Description
 
-`.dvc` and `dvc.lock` [files](/doc/user-guide/dvc-files-and-directories) act as
-pointers to specific version of data files or directories tracked by DVC. This
-command synchronizes the workspace data with the versions specified in the
-current `.dvc` and `dvc.lock` files.
+[`dvc.lock`](/doc/user-guide/dvc-files-and-directories#dvclock-file) and
+[`.dvc`](/doc/user-guide/dvc-files-and-directories#dvc-files) files act as
+pointers to the <abbr>cached</abbr> contents of data tracked by DVC. This
+command synchronizes the workspace data with the tracked file contents specified
+in the current `.dvc` and `dvc.lock` files.
 
-`dvc checkout` is useful, for example, when using Git in the
-<abbr>project</abbr>, after `git clone`, `git checkout`, or any other operation
-that changes the DVC files in the workspace.
+`dvc checkout` is useful, for example, after `git clone`, `git checkout`, or any
+other operation that changes the DVC files in the <abbr>workspace</abbr> (when
+using Git in the <abbr>project</abbr>). This allows to restore different
```

Comment:
let's not complicate this ... how is it relevant to the PR?

Reply:
Its just a re-phrasing of the existing sentence. Easier to read IMO. But OK, rolled it back to "...is useful, for example, when using Git in the project, after `git clone`..."

The "This allows to restore different versions of DVC-tracked files and directories." note is pretty important now I think (differentiation between tracking and versioning), kind of missing from this ref.